### ISSUE TITLE
Feature Request - Indoor Humidity

### ISSUE THREAD

**Issue by mrxaero** [poblabs/weewx-belchertown#84]
I would like to propose a feature request/enhancement of displaying the indoor humidity. This would be an option just like enabling indoor temperature within the skin.conf.

**Comment by poblabs** [comment]
Skin version 1.0 will have this ability - as well as the ability to customize any chart and show any data you want. It's under development right now with a beta test out in a few weeks. 

For example to show indoor humidity you would use this config:

```
        [[[testchart]]]
            title = Indoor Humidity
            [[[[inHumidity]]]]
```

and it would show this chart:

![image](https://user-images.githubusercontent.com/3484775/54820828-d2eb5200-4c76-11e9-907c-18a82dc4c013.png)

**Comment by mrxaero** [comment]
Could this be combined with the outdoor humidity graph instead of it being its own?

**Comment by poblabs** [comment]
Yeah, you can even use different chart types, too. Again, this won't be ready until 1.0. 

Standard line chart

```
        [[[testchart]]]
            title = Humidity
            [[[[inHumidity]]]]
            [[[[outHumidity]]]]
```

![image](https://user-images.githubusercontent.com/3484775/54821494-b2bc9280-4c78-11e9-9abe-fc6cf7f719ab.png)

Outdoor is a line chart, indoor is an area chart

```
        [[[testchart]]]
            title = Humidity
            [[[[inHumidity]]]]
                type = area
            [[[[outHumidity]]]]
```

![image](https://user-images.githubusercontent.com/3484775/54821506-b9e3a080-4c78-11e9-9d19-5401329ad9d7.png)


Column chart

```
        [[[testchart]]]
            title = Humidity
            type = column
            [[[[inHumidity]]]]
            [[[[outHumidity]]]]
```

![image](https://user-images.githubusercontent.com/3484775/54821511-bf40eb00-4c78-11e9-8a0b-d86d32e7ad7b.png)


You could even add temperature to the humidity chart and assign the temperature to the right yAxis (`yAxis = 1`)

```
        [[[testchart]]]
            title = Humidity
            [[[[inHumidity]]]]
            [[[[outHumidity]]]]
            [[[[inTemp]]]]
                yAxis = 1
            [[[[outTemp]]]]
                yAxis = 1
```

![image](https://user-images.githubusercontent.com/3484775/54821586-05964a00-4c79-11e9-9d53-13f16b05f629.png)


Want to go crazy and have wind on it too? Sure. 

```
        [[[testchart]]]
            title = Humidity
            [[[[inHumidity]]]]
            [[[[outHumidity]]]]
            [[[[inTemp]]]]
                yAxis = 1
            [[[[outTemp]]]]
                yAxis = 1
            [[[[windSpeed]]]]
                yAxis = 1
            [[[[windGust]]]]
                yAxis = 1
```

![image](https://user-images.githubusercontent.com/3484775/54821645-39716f80-4c79-11e9-8d92-0d4cd2cbaa97.png)

**Comment by slvrscoobie** [comment]
Oh this is exactly what i need! I have 8 indoor sensors I want to be able to put all together with the outdoor information as well!

**Comment by poblabs** [comment]
This will be available in skin 1.0
